### PR TITLE
chore(flat-components): generate "export *"

### PR DESCRIPTION
**Pull request by hyrious**

### CODE REVIEW

**Review by crimx — APPROVED** [review]

File: packages/flat-components/rollup.config.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ function generateIndexTS(compNames) {
 
     const str =
         "// THIS IS AN AUTO-GENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.\n\n" +
-        compNames.map(name => `export { ${name} } from "./components/${name}";`).join("\n") +
```
Comment (BlackHole1) [inline packages/flat-components/rollup.config.js:124]: 这里改动的原因是什么

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

File: packages/flat-components/rollup.config.js (hunk 1 above)
Comment (crimx) [reply packages/flat-components/rollup.config.js:124]: 扁平化 export

Comment (BlackHole1) [reply packages/flat-components/rollup.config.js:124]: 这样做的话，会不会出现，无法 tree shaking

Comment (hyrious) [reply packages/flat-components/rollup.config.js:124]: 不会，可以看这个[例子](https://rollupjs.org/repl/?version=2.42.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMGElMjAlN0QlMjBmcm9tJTIwJTVDJTIyLiUyRmEuanMlNUMlMjIlNUNuY29uc29sZS5sb2coYSklMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJhLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMColMjBmcm9tJTIwJTVDJTIyLiUyRmFhLmpzJTVDJTIyJTIyJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMmFhLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGNvbnN0JTIwYSUyMCUzRCUyMDQyJTVDbmV4cG9ydCUyMGNvbnN0JTIwYiUyMCUzRCUyMDQyJTIyJTdEJTVEJTJDJTIyb3B0aW9ucyUyMiUzQSU3QiUyMmZvcm1hdCUyMiUzQSUyMmVzJTIyJTJDJTIybmFtZSUyMiUzQSUyMm15QnVuZGxlJTIyJTJDJTIyYW1kJTIyJTNBJTdCJTIyaWQlMjIlM0ElMjIlMjIlN0QlMkMlMjJnbG9iYWxzJTIyJTNBJTdCJTdEJTdEJTJDJTIyZXhhbXBsZSUyMiUzQW51bGwlN0Q=)

**Review by BlackHole1 — APPROVED** [review]

**Pull request merged**